dispatcher/frontend-ui/src/constants.js

```diff
@@ -322,17 +322,17 @@ export default {
   cancelable_statuses: cancelable_statuses,
   running_statuses: running_statuses,
   contact_email: "contact@kiwix.org",
-  categories: ["gutenberg", "ifixit", "other", "phet", "psiram", "stack_exchange",
+  categories: ["freecodecamp", "gutenberg", "ifixit", "other", "phet", "psiram", "stack_exchange",
                "ted", "openedx", "vikidia", "wikibooks", "wikihow", "wikinews",
                "wikipedia", "wikiquote", "wikisource", "wikispecies", "wikiversity",
                "wikivoyage", "wiktionary"],  // list of categories for fileering
-  warehouse_paths: ["/gutenberg", "/ifixit", "/other", "/phet", "/psiram", "/stack_exchange",
+  warehouse_paths: ["/freecodecamp", "/gutenberg", "/ifixit", "/other", "/phet", "/psiram", "/stack_exchange",
```

Comment:
Also for @kelson42 to decide on but I believe those FCC ZIMs will fall under `other` given there's gonna be only one for now. I don't think it matches `mooc` either.